api/client.go

```diff
@@ -93,16 +138,12 @@ func (c *Config) ReadEnvironment() error {
 	var envCAPath string
 	var envClientCert string
 	var envClientKey string
-	var envInsecure bool
-	var foundInsecure bool
+	envInsecure := true
```

Comment:
Yes, should definitely be false. Also, as a coding style thing, please just use `var envInsecure bool` to match the rest.